README.rst

```diff
@@ -207,16 +205,19 @@ the computational workflow steps and the expected outputs:
     outputs:
       files:
        - outputs/plot.png
-    environments:
-      - type: docker
-        image: reanahub/reana-env-jupyter
     workflow:
-      type: yadage
-      file: workflow/yadage/workflow.yaml
+      type: serial
+      specification:
+        steps:
+          - environment: 'reanahub/reana-env-jupyter'
+            commands:
+              - mkdir -p outputs
```

Comment:
Isn't this a bit too little to spawn a new container to execute it?

Reply:
Yes, we can put in all in one single command... I'll joint it.

It would be good to allow for whitespace and newlines to have a nicer `reana.yaml`.  A quick test with `>` wasn't successful, but we can look at that later...